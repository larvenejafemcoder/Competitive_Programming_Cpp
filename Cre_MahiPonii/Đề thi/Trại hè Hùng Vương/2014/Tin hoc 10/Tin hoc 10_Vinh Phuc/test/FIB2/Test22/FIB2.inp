79
41 169042783
17 2435
35 1026528
6 13
20 9447
9 40
5 4
24 47667
32 52040
25 88020
0 1
28 388920
25 26395
42 170113099
7 12
17 933
30 1201240
29 10820
39 49703538
29 298945
12 142
24 65959
23 2145
45 1696307561
22 21265
7 3
7 21
35 6465000
11 41
17 2018
38 51953666
0 1
28 336286
5 1
19 4537
38 37212951
32 2381763
19 711
43 625058438
9 54
29 175841
27 189903
11 124
6 13
40 4068830
41 265589356
29 92760
24 23423
29 189251
9 11
43 70465315
43 351059389
13 245
16 1502
13 280
10 78
30 400642
7 5
16 811
41 81578675
45 235074653
30 1273757
36 10803644
10 88
40 133296823
32 812424
45 812004345
29 172311
8 16
13 16
11 100
29 68205
14 478
45 1564867216
24 52760
42 59993721
11 7
26 32181
32 1047399
