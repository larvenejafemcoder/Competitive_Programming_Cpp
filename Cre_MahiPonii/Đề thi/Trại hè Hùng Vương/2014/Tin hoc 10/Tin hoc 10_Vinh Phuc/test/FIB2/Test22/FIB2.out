64568598
930
392099
5
3609
15
2
18207
19878
33621
1
148554
10082
64977422
5
356
458833
4133
18985062
114187
54
25194
819
647931833
8123
1
8
2469410
16
771
19844535
1
128450
0
1733
14214083
909753
272
238751078
21
67165
72537
47
5
1554155
101446107
35431
8947
72288
4
26915355
134092755
94
574
107
30
153032
2
310
31160281
89790528
486532
4126625
34
50914856
310318
310158061
65817
6
6
38
26052
183
597726089
20153
22915562
3
12292
400071
